react/AppTile/index.jsx

```diff
@@ -52,14 +55,30 @@ export const AppTile = ({
     ? showStatus.indexOf(statusLabel) > -1 && statusLabel
     : showStatus && statusLabel
   IconComponent = IconComponent || AppIcon
+  const isInMaintenance = maintenanceDisplay && statusLabel === 'maintenance'
+
   return (
-    <Tile tag="button" type="button" onClick={onClick}>
+    <Tile
+      tag="button"
+      type="button"
+      onClick={onClick}
+      className={
+        isInMaintenance ? styles['AppTile-container-maintenance'] : undefined
+      }
+    >
       <TileIcon>
         <IconComponent
           app={app}
           className={styles['AppTile-icon']}
           {...getAppIconProps()}
         />
+        {isInMaintenance && (
+          <Icon
+            data-testid="icon-maintenance"
+            icon={WrenchCircleIcon}
+            className={`${styles['AppTile-icon-maintenance']} u-coolGrey`}
```

Comment:
Why this `u-coolGrey`? By default `<Icon` will use `icon` classname https://github.com/cozy/cozy-ui/blob/master/react/Icon/index.jsx#L58 which is https://github.com/cozy/cozy-ui/blob/master/react/Icon/styles.styl#L4 which is 

`currentColor` https://github.com/cozy/cozy-ui/blob/master/stylus/settings/palette.styl#L246 

So: 
- Maybe Icon shouldn't be colored by default by currentColor but by secondaryTextColor ? (cc @joel-costa) (Like we do for invertedThem) 
- let's use the color props instead of an utility class to be more declarative ;)